lib/tasks/attempts.rake

```diff
@@ -1,13 +1,15 @@
 namespace :attempts do
-  auth_token = 'abc123'
+  auth_token = IdentityConfig.store.irs_attempt_api_auth_tokens.sample
```

Comment:
This was not strictly related to the PR, but hopefully can make the script ever so slightly easier to use

Reply:
🤔  Do we need to put this inside the task so that the environment (and config/secrets) are already loaded?

Reply:
huh, that's a good point that I hadn't noticed.  it does work in my testing though

Reply:
I guess we have `IdentityConfig` set up to be used before the rails application loads so it's probably fine